website/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+gem "github-pages", "227"
```

Comment:
Should we also fix the Ruby version used? See https://bundler.io/guides/gemfile_ruby.html#specifying-a-ruby-version.

Reply:
Would like to make part of the larger initiative, see https://github.com/PicnicSupermarket/error-prone-support/pull/253#issuecomment-1265019555

Reply:
Sorry, I didn't read all of the comments so was just a drive-by comment. SGTM 👍